src/rt/cover.d

```diff
@@ -115,6 +117,32 @@ static ~this()
     const NUMLINES = 16384 - 1;
     const NUMCHARS = 16384 * 16 - 1;
 
+    // Only use with string literals or make sure name ends with \0
+    static char[] getenv(char[] name, char[] default_ = null)
+    {
+        auto c_val = cstdlib.getenv(name.ptr);
+        if (c_val is null)
+            return null;
+        auto val = c_val[0..cstring.strlen(c_val)];
+        if (val is null)
+            val = default_;
+        return val;
+    }
+
+    // Get runtime options from environment variables
+    auto merge_str = getenv("DRT_COVMERGE");
+    if (merge_str !is null) switch (merge_str)
+    {
+        case "0": merge = false; break;
+        case "1": merge = true;  break;
+        default:
+            fprintf(stderr, "RT Error: Runtime option " ~
+                    "'DRT_COVMERGE' should be 1 or 0, got '%.*s'.\n".ptr,
+                    cast(int)merge_str.length, merge_str.ptr);
+    }
+    srcpath = getenv("DRT_COVSRCPATH");
```

Comment:
```
srcpath = getenv("DRT_COVSRCPATH", srcpath);
```
for more robust code?

Reply:
Damn! Yeah, the whole point of using `default_` ( :stuck_out_tongue_closed_eyes: ) was to use it here!